source/agora/common/Database.d

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+
+    Contains a managed wrapper around SQLite databases.
+
+    This class ensures that all D2SQLite databases handled by the thread
+    are cleanly destroyed after the thread shuts down.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.common.Database;
+
+import SQLite = d2sqlite3.database;
+import d2sqlite3.sqlite3;
+
+/// Ditto
+public class Database
```

Comment:
Consider renaming this to something like `ManagedDatabase`. We don't want to accidentally instantiate the wrong type of database because of a wrong import (d2sqlite vs agora import).